lib/Music/music_album_manager.rb

```diff
@@ -42,7 +61,10 @@ def create_music_album
     genre = Genre.new(genre_name)
     music_album.genre = genre
     @albums << music_album
-    puts "Music Album [Genre: \"#{music_album.genre.name.upcase}\", Published Date: #{music_album.publish_date}, On Spotify?: #{music_album.on_spotify ? 'Yes' : 'No'}] Created Successfully!"
```

Comment:
Hello @batoolfatima2135, Kindly add this code again because I was printing the detailed information of the album after created successfully. Thank you for your cooperation. 😁